docs/tasks/manage-stateful-set/scale-stateful-set.md

```diff
@@ -0,0 +1,83 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+
+---
+
+{% capture overview %}
+This page shows how to scale a Stateful Set.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* Stateful Sets are only available in Kubernetes release >= 1.5. 
+* Stateful Sets are previously known as Pet Sets in Kubernetes release 1.3-1.4. You can either upgrade your Pet Sets to Stateful Sets, 
+or just change all `statefulset` references to `petset`. *TODO: link to upgrade from Pet Sets to Stateful Sets.*
+* **Not all stateful applications scale nicely.** You need to understand your Stateful Sets well before continue. If you're unsure, remember that it may not be safe to scale your Stateful Sets. 
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Use `kubectl` to scale Stateful Sets
+
+#### `kubectl scale` (>= 1.4 release)
+
+First, find the Stateful Set you want to scale. Remember, you need to first understand if you can scale it or not. 
+
+```shell
+kubectl get statefulsets <stateful-set-name>
+```
+
+If you wish to change the number of replicas of your Stateful Set, just use this command:
+
+```shell
+kubectl scale statefulsets <stateful-set-name> --replicas=<new-replicas>
+```
+
+Note that `kubectl scale` only works on Stateful Set with Kubernetes release >= 1.4. 
```

Comment:
The current doc plan is to deprecate the old user-guide for PetSets and link to StatefulSet concept guide there.